src/responses/yuzu.json

```
@@ -45,7 +45,12 @@
       "raptor": { "reply": "Raptor is a 3rd party service that provides a paid alternative to Nintendo Switch Online. We are not associated with Raptor in anyway. After key emulation members and the discerning public shared their opinion with us, we cut all ties with Raptor immediately in 2020. This new fork was made without our permission or cooperation and does not have permission to use the \"yuzu\" name or branding. At this time, we have no plans to add support for any 3rd party services, including Raptor.\n\nAs per our normal policies, we only provide support for our official releases and discussion of any unofficial builds isn't allowed."},
       "role": { "reply": "To claim your Patreon Discord role, please follow this guide: <https://support.patreon.com/hc/en-us/articles/212052266-Get-my-Discord-role>"},
       "release": { "reply": "yuzu builds can be manually downloaded on Github: <https://github.com/yuzu-emu/yuzu-mainline/releases>"},
-      "rec": { "reply": "For information on recommended settings and GPU drivers for yuzu, please refer to this page: <https://community.citra-emu.org/t/recommended-settings/319349>"}
+      "rec": { "reply": "For information on recommended settings and GPU drivers for yuzu, please refer to this page: <https://community.citra-emu.org/t/recommended-settings/319349>"},
+      "drivers": { "reply": "**Custom Drivers:** \nIf you have a device with a Qualcomm processor, you can explore the option of using custom drivers to potentially enhance performance and compatibility.\n\nPlease make sure to obtain the appropriate drivers for your specific device in the #android-drivers channel.\nFor Snapdragon 8 Gen1 or newer devices, you will need the Adreno 7xx drivers. For other devices, you will need the a6xx drivers."},
+      "ram": { "reply": "**Android RAM Requirements:** \nTo play most games on Yuzu for Android, a minimum of 8GB of RAM is currently required. While 6GB of RAM may work, it is only suitable for simple or small games"},
+      "powervr": { "reply": "**Unsupported PowerVR GPU** \nRegrettably, your device is equipped with a PowerVR GPU, which lacks the necessary features for us to provide support."},
```

```suggestion
      "powervr": { "reply": "**Unsupported PowerVR GPU** \nYour device is equipped with a PowerVR GPU, which lacks the necessary features for us to provide support."},
```
most our users are non-english, so let's try to make it easy to understand for them